### ISSUE TITLE
Please make a function in PTU to get 'AZ' and 'EL' angles.

### ISSUE THREAD

**Issue by bvnayak** [nikhilkalige/flir_ptu#1]
We have `put.pan()` and `pt.tilt()` which returns PP and TP values.
Please make similar function/methods which allows user to get back `AZ` and `EL` values.
e.g.

```
az = ptu.az
el = ptu.el
```

**Comment by nikhilkalige** [comment]
Use `pan_angle()` and `tilt_angle` for that

**Comment by bvnayak** [comment]
```
In [6]: a = ptu.pan_angle()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-6-19b7775e2277> in <module>()
----> 1 a = ptu.pan_angle()

/home/pi/workspace/flir_ptu/flir_ptu/ptu.py in pan_angle(self, angle_value)
    142         else:
    143             data = self.pan()
--> 144             return data * (92.5714/3600)
    145
    146     def tilt_angle(self, angle_value=False):

TypeError: can't multiply sequence by non-int of type 'float'
```

**Comment by nikhilkalige** [comment]
Fixed with c775508
